controllers/events_controller.js

```diff
@@ -411,12 +465,12 @@ module.exports.updateEvent = async (req, res) => {
 			isRegistrationRequired
 		};
 
-		if (req.files) {
+		if (req.files.length !== 0) {
```

Comment:
can be the case when req.files is undefined, it will give cannot find property error!
possible: if(req.files && req.files.length !== 0)

Reply:
Cul, I'll change it